src/environments/environment.ts

```diff
@@ -3,7 +3,8 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: false
+  production: false,
+  publicStorageAccount: 'https://sunbirddev.blob.core.windows.net/'
```

Comment:
This we should take it from Devops environment variables. So that this can be changed for Dev & Staging.